contrib/epee/include/net/abstract_tcp_server2.h

```diff
@@ -213,6 +213,8 @@ namespace net_utils
 
     int get_binded_port(){return m_port;}
 
+    long get_connections_count(){return (m_sock_count - 1);} // Socket count minus listening socket
```

Comment:
const

Reply:
Gotcha, `const` in this case since the return value isn't going to be modified. Is there a reason why the `get_binded_port` above isn't using `const` here? Just trying to understand why one would need it but the other not when they both appear to have the same sort of usage (returning data that won't change in the context of it being fetched).

Reply:
I don't know why. Presumably the author wasn't a const stickler.

Reply:
Gotcha, that's fine I was just curious as I make attempts to be consistent with existing code. I'll use const where appropriate going forward over that consistency.

Reply:
I got the following warning: `'const' type qualifier on return type has no effect`

Thinking it over I guess the only way to have this be const would be to pass it back by reference since if it's an long it'll just copy it by value so the const isn't doing anything in this case.

If I didn't need to subtract the listener socket from `m_sock_count` I could just return that reference and then it could be const. 

That may be why the other method isn't `const` as well since it would just throw warnings and not guarantee someone couldn't change the return value. I could still add it if we want auto complete etc to show that the value shouldn't change (gives callers of the method a more explicit interface in terms of what our expectations are).

Reply:
That's probably because you're doing something like:
const int get_foo()
but you need to do:
int get_foo() const

Reply:
^ Yup! That's exactly what I had done. Switching it to that approach does not cause any warnings with the compilation. 

Thanks for the help! Will keep this syntax in mind with future pull requests that return const data. :+1:

Reply:
Probably an oversight. Can this value ever be zero? Does a check needed to be added here for that?

Reply:
If this was called prior to the server's `init_server` call then it could return -1 as the initial listening socket wouldn't have been created yet.  
https://github.com/monero-project/monero/blob/master/contrib/epee/include/net/http_server_impl_base.h#L76
https://github.com/monero-project/monero/blob/master/contrib/epee/include/net/abstract_tcp_server2.inl#L733

From an RPC call perspective it'll always be at least 2 here prior to the subtraction (listening socket and the connection that triggered the RPC method) but it's not crazy to think this code may be called outside of that.

It's easy enough to add a quick check though and return 0 if the value is 0 (not initiated) and it makes this interface more consistent / removes the possibility of someone using this prior to initialization and getting odd results.